src/components/FilterTop.js

```diff
@@ -100,6 +100,9 @@ function Filter(props) {
   const [organizationId, setOrganizationId] = useState(
     filter.organizationId || ALL_ORGANIZATIONS
   );
+  const [stakeholderUUID, setStakeholderUUID] = useState(
```

Comment:
We really need to try to combine the two filter components into a common component.
The amount of duplication and risk of mistakes is getting silly.

Reply:
Yeah, now that we have more of an idea what each tool is filtering we can probably create a FilterHeader and a particular filter forms for the unique fields: Verify, Capture, Grower.  But certain fields might be broken out into their own components -- like the org selection, species selection, verify status

What do you think?